ops-bedrock/docker-compose-testnet.yml

```diff
@@ -0,0 +1,183 @@
+version: '3.4'
+
+# This Compose file is expected to be used with the testnet-up.sh script.
+# The volumes below mount the configs generated by the script into each
+# service.
+
+volumes:
+  l1_data:
+  l2_data:
+  op_log:
```

Comment:
We'd name our volume here so it's easier to manage. Perhaps something for later.